trufflehog-ignore.txt

```diff
@@ -0,0 +1,4 @@
+package.json
+package-lock.json
+RELEASE.md
```

Comment:
Why are release note and readme being ignored?

Reply:
Because they contain things like github.com/quantumblacklabs/blob/<high-entropy-string> :(